docs/release.md

```diff
@@ -10,11 +10,18 @@ git push -u upstream release-0.1
 ```
 
 ```bash
+# Export the tag of the release to be cut, e.g.:
 export RELEASE_TAG=v0.1.1
 
+# Create tags locally
+# Warning: The test tag MUST NOT be an annotated tag.
 git tag -s -a ${RELEASE_TAG} -m ${RELEASE_TAG}
+git tag test/${RELEASE_TAG}
```

Comment:
let's sign this tag with gpg signature as above?

Reply:
This process is coming from CAPI release document. They explicitly don’t sign test package. I’m not sure why exactly, but I’d rather stick with existing process for now: https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#repeatedly-cut-a-release-1

Reply:
I wonder why, though, but it is fine. Thanks!

/lgtm